Makefile-tests.am

```diff
@@ -121,6 +121,7 @@ _installed_or_uninstalled_test_scripts += \
 	tests/test-refs-collections.sh \
 	tests/test-remote-add-collections.sh \
 	tests/test-summary-collections.sh \
+	tests/test-pull-collections.sh \
```

Comment:
I think it makes sense to keep the tests as a separate commit, since they test all the functionality added in the previous commits.